projects/novo-elements/src/elements/form/Control.spec.ts

```diff
@@ -71,7 +71,7 @@ describe('Test Localization', () => {
 
   it('should set decimal separator based on locale correctly', () => {
     const component = new NovoControlElement(mockElement, null, null, null, null, null, 'fr-FR');
-    expect(component.decimalSeparator).toBe('.');
+    expect(component.decimalSeparator).toBe(',');
```

Comment:
I approve of the changes overall - I'm just not sure if this is going on here - is this a case where the test is broken on master, or just a branch? Is this a case where the decimal separator is actually being based on the real physical locale so that it works in some geographic locations but not others?

Reply:
The unit test appears to be broken on master, i assumed the locale was being passed in as the last arg in line 73 and looks like France uses comma as a decimal separator vs a period.

Reply:
Perfect. thanks!